packages/dos-donts-block/src/types.ts

```diff
@@ -77,6 +77,11 @@ export type SortableDoDontItemProps = {
     doColor: Color;
     dontColor: Color;
     onChangeItem: (id: string, value: string | number | undefined, type: 'title' | 'body' | 'type' | 'imageId') => void;
+    onChangeLocalItem: (
+        id: string,
+        value: string | number | undefined,
+        type: 'title' | 'body' | 'type' | 'imageId'
+    ) => void;
```

Comment:
```suggestion
    onChangeLocalItem: (id: string, value: ValueType, type: ChangeType) => void;
```

Also on the other types

Reply:
Thanks, fixed!